tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -373,6 +384,13 @@ func (az *AzureProvider) createNIC(ctx context.Context, vnetName, subnetName, ns
 		fmt.Errorf("failed to get ip address: %v", err)
 	}
 
+	// In Azure, for egress as well as for Load balancer service to work, the
+	// windows vm is expected to be placed behind the worker nodes' Load balancer.
```

Comment:
`Windows VM`
`load balancer`

Please be consistent. Same issue in your commit messages.